admin/metabox/class-metabox.php

```diff
@@ -571,6 +571,14 @@ public function do_meta_box( $meta_field_def, $key = '' ) {
 
 		switch ( $meta_field_def['type'] ) {
 			case 'pageanalysis':
+			    $options = $this->options;
+				$content_analysis_active = $options['content_analysis_active'];
+				$keyword_analysis_active = $options['keyword_analysis_active'];
```

Comment:
I would use $this->options directly, because saving that value to $options isn't necessary.